ibis/backends/pandas/execution/maps.py

```diff
@@ -167,19 +168,20 @@ def execute_map_keys_series(op, data, **kwargs):
 def execute_map_keys_dict(op, data, **kwargs):
     if data is None:
         return None
-    return list(data.keys())
+    return np.array(list(data.keys()))
 
 
 @execute_node.register(ops.MapValues, pd.Series)
 def execute_map_values_series(op, data, **kwargs):
-    return data.map(safe_values)
+    res = data.map(safe_values)
+    return res
 
 
 @execute_node.register(ops.MapValues, (collections.abc.Mapping, type(None)))
 def execute_map_values_dict(op, data, **kwargs):
     if data is None:
         return None
-    return list(data.values())
+    return np.array(list(data.values()))
```

Comment:
Do we need the list here?

Reply:
Yes I think so—`list(...)` is used to unpack the iterable (see [the other comment](https://github.com/ibis-project/ibis/pull/2753#discussion_r625373706))